### PR TITLE
change component name

### PR DESCRIPTION
**Pull request by D-W-**
# PR into Azure/azureml-examples

## Checklist

I have:

- [ ] read and followed the contributing guidelines

## Changes

-

fixes #

### CODE REVIEW

**Comment by elliotzh** [comment]
Is this just to unify style or guarantee consistency as we will do lower in client?

File: cli/jobs/pipelines-with-components/basics/1b_e2e_registered_components/train.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
-name: Train
+name: train
```
Comment (wangchao1230) [inline cli/jobs/pipelines-with-components/basics/1b_e2e_registered_components/train.yml:2]: will this break ci?

**Comment by wangchao1230** [comment]
rename your branch to get ci running

File: .github/workflows/cli-jobs-pipelines-cifar-10-pipeline.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@ on:
     - cron: "0 0/4 * * *"
   pull_request:
     branches:
+      - march-cli-preview
```
Comment (elliotzh) [inline .github/workflows/cli-jobs-pipelines-cifar-10-pipeline.yml:8]: should we keep them here?

**Review by elliotzh — APPROVED** [review]

File: .github/workflows/cli-jobs-pipelines-cifar-10-pipeline.yml (hunk 2 above)
Comment (D-W-) [reply .github/workflows/cli-jobs-pipelines-cifar-10-pipeline.yml:8]: As last discussion, we plan to add gate for this branch?

**Review by cloga — APPROVED** [review]

**Pull request merged**